index.html

```diff
@@ -134,7 +134,7 @@ <h5>Actors</h5>
             </p>
           </form>
           <br>
-          <a class="waves-effect waves-light btn">Show Me The Movies!</a>
+          <button id="suggestions-button"  class="waves-effect waves-light btn">Show Me The Movies!</button>
```

Comment:
Please use the button ID that is already in master, this button is being called in another function.